File: app/src/main/java/com/example/treediary/MainActivity.java
package com.example.treediary;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.RequiresPermission;
import androidx.appcompat.app.AppCompatActivity;

import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void onButtonClicked(View view) {
        Intent intent = new Intent( getApplicationContext(), explain_page.class);
        startActivity(intent);
    }

    public void writeButton(View view) {
        Intent intent = new Intent( getApplicationContext(), CreateNewDiary.class);
        startActivity(intent);
    }

    public void listButton(View view) {
        Intent intent = new Intent( getApplicationContext(), ListOfDiaryActivity.class);
        startActivity(intent);
    }

}

File: app/src/main/java/com/example/wishwish/MainActivity.kt
package com.example.wishwish

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val question: Button = findViewById(R.id.question_button)
        val write: Button = findViewById(R.id.write_button)
        val list: Button = findViewById(R.id.list_button)
        val maintree : ImageView = findViewById(R.id.main_tree)

        question.setOnClickListener {
            val intent = Intent(this, ExplainActivity::class.java)
            startActivity(intent)
        }

        write.setOnClickListener {
            val intent = Intent(this, DiaryActivity::class.java)
            startActivity(intent)
        }

        list.setOnClickListener {
            val intent = Intent(this, ListActivity::class.java)
            startActivity(intent)
        }

        maintree.setOnClickListener {
            val intent = Intent(this, HappyActivity::class.java)
            startActivity(intent)
        }
    }
}